velox/exec/Task.h

```diff
@@ -498,6 +498,14 @@ class Task : public std::enable_shared_from_this<Task> {
   /// during this wait call. This is for testing purpose for now.
   static void testingWaitForAllTasksToBeDeleted(uint64_t maxWaitUs = 3'000'000);
 
+  void setSpillDir(const std::string& spillDir) {
```

Comment:
Using setters is an anti-pattern. Can we pass it to the Task's constructor?

Reply:
@mbasmanova 
Yes we can change this, although I strongly disagree with 'anti-pattern' message here. It is very subjective. :)